packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -494,7 +494,10 @@ export class ScheduleTrigger implements INodeType {
 
 			if (interval[i].field === 'hours') {
 				const hour = interval[i].hoursInterval as number;
-				const minute = interval[i].triggerAtMinute?.toString() as string;
+				let minute = interval[i].triggerAtMinute?.toString() as string;
+				if (!minute && nodeVersion >= 1.2) {
```

Comment:
Does this need a node version? I don't think users ever intend to run every minute when they configure hourly

Reply:
I think it's better to use version as without this behavior of some workflow that are currently running in production would change